src/transformer/utils/extensions/jisc.php

```diff
@@ -27,7 +28,7 @@ function jisc(array $config, \stdClass $event, $course) {
                 'http://id.tincanapi.com/extension/ip-address' => $event->ip,
                 'http://xapi.jisc.ac.uk/statementCat' => 'VLE',
             ],
-            course_area($course)
+            extensions\course_area($course)
         );
```

Comment:
Can't we just do \utils\extensions\course_area and drop the additional use statement on line 21?

Reply:
I added it to stay consistent with the style used in the rest of the plugin.

Reply:
I've not found `use src\transformer\utils\extensions as extensions;` anywhere else in the codebase.

If we are going for consistency I think it should be `utils\extensions\course_area`, in a similar way it has been done in admin/tool/log/store/xapi/src/transformer/events/core/course_completed.php: 
`'object' => utils\get_activity\course($config, $course)`

Either that, or we drop the changes from this file (and base.php) altogether as they are not related to the ticket